.yamllint

```diff
@@ -0,0 +1,10 @@
+---
```

Comment:
to avoid cluttering the root directory with dot files, what about moving this to the `tests` dir?